profile.md

```diff
@@ -508,6 +508,12 @@ in the `metadata` objects, it is RECOMMENDED that the following names
 be used when possible in an attempt to provide some degree of interoperability
 and consistency.
 
+Please note that in addition service metadata returned from [catalog endpoint](spec.md#catalog-management), 
+Service Brokers MAY also expose service-instance-specific metadata through the
+[ServiceInstanceMetadata](spec.md#service-instance-metadata) structure returned by Service Brokers 
+in the [Provisioning](spec.md#provisioning) endpoint with updates potentially published 
```

Comment:
Could you add Update here too? ServiceInstanceMetadata can be provided as a response to an update request https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#body-7

Reply:
Good catch, thanks. Fixed in 284b310a5ddb9da1d7dd00066caecfad5c01c199

Reply:
Thanks!